### PR TITLE
[Prometheus] add taint toleration to server

### PR DESCRIPTION
**Pull request by yagonobre**

### CODE REVIEW

**Comment by yagonobre** [comment]
up cla

**Comment by dhilipkumars** [comment]
@yagonobre Please fix the lint error

**Comment by yagonobre** [comment]
Done!

File: stable/prometheus/values.yaml
Comment (dhilipkumars) [inline stable/prometheus/values.yaml:338]: Could you add suitable example configuration like below?
```
tolerations: []
  #- key: "key"
  #operator: "Equal|Exists"
  #value: "value"
  #effect: "NoSchedule|PreferNoSchedule|NoExecute(1.6 only)"
```

Please take a look at this `./stable/traefik/values.yaml` for reference

**Review by dhilipkumars — CHANGES_REQUESTED** [review]
Looks good to me, one minor change request

File: stable/prometheus/values.yaml
Comment (yagonobre) [reply stable/prometheus/values.yaml:338]: Just a minute

Comment (dhilipkumars) [reply stable/prometheus/values.yaml:338]: Apologies for not explaining clearly the first time, i meant the commented defaults which might also explain about possible combinations the user can choose from. 
```
tolerations: []
  #- key: "key"
  #operator: "Equal|Exists"
  #value: "value"
  #effect: "NoSchedule|PreferNoSchedule|NoExecute(1.6 only)"
```
Could you update as above instead? Also while you are adding toleration for this if you could add same example to the `toleration` in `node-exporter ` that would be great.  Otherwise LGTM from myside

Comment (yagonobre) [reply stable/prometheus/values.yaml:338]: Done!

**Review by dhilipkumars — APPROVED** [review]

**Comment by dhilipkumars** [comment]
LGTM from my side 
@prydonius could you PTAL and approve if all is okay?

**Comment by dhilipkumars** [comment]
cc: @kubernetes/charts-maintainers

File: stable/prometheus/README.md
Comment (unguiculus) [inline stable/prometheus/README.md]: Remove backtick after `server`

File: stable/prometheus/templates/server-deployment.yaml
Comment (unguiculus) [inline stable/prometheus/templates/server-deployment.yaml]: `nodeTolerations` -> `tolerations`

**Review by unguiculus — CHANGES_REQUESTED** [review]

**Review by unguiculus — APPROVED** [review]

**Pull request merged**
